### ISSUE TITLE
oelint.vars.dependsappend should only after an include or inherit statement

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#286]
The outcome of https://lists.openembedded.org/g/openembedded-architecture/topic/87569589#1372 is that

```
DEPENDS = "a"

inherit something
```

is a valid statement, while

```
inherit something

DEPENDS = "a"
```

or

```
require/include something.inc

DEPENDS = "a"
```

should still trigger an oelint.vars.dependsappend warning
